frontend/src/app/shared/components/modal/modal-overlay.component.html

```diff
@@ -0,0 +1,19 @@
+<div
+  class="spot-modal-overlay"
+  [class.spot-modal-overlay_active]="(activeModalInstance$ | async) !== null"
+  [class.spot-modal-overlay_not-full-screen]="notFullscreen"
```

Comment:
just to satisfy my interest: does this syntax has any functional advantages over using `[ngClass]`?

Reply:
It should not make any difference, however it was changed due to change detection not correctly firing on ngClass. I'm not sure why, but there is an older bug in Angular that might be related to that (and would be fixed by upgrading)